app/services/set_school_local_authority_district.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class SetSchoolLocalAuthorityDistrict < BaseService
+  def call
+    if school.local_authority_district&.code != administrative_district_code
+      ActiveRecord::Base.transaction do
+        school.school_local_authority_districts.latest.first&.update!(end_year: start_year)
```

Comment:
Pretty inconsistent with LA

Reply:
We should change the `latest` scope from a `where` to `find_by`. I know I wrote it, but it doesn't make any sense

Reply:
@ethanmills it's not the best idea to use `find_by` in a scope as it fires a db query. Scopes should only return scopes (using `where`, `order` and other methods that actually return scopes).
I might be missing some context here, looks `latest` could be better served by a method that executes the query or by a scope using `order` + `limit`.